bokeh/core/property/text_like.py

```diff
@@ -41,7 +41,14 @@
 # General API
 #-----------------------------------------------------------------------------
 
-TextLike = Either(MathString, Instance("bokeh.models.text.BaseText"))
+class TextLike(Either):
+    """ Accept a string that may be interpreted into text models or the models themselves.
+
+    """
+
+    def __init__(self, default=None, help=None) -> None:
```

Comment:
Are you sure `None` is the right default value for effectively `String | Instance(...)` type? This should be either `Undefined`, if you want to force a default definition at use-sites, or `Intrinsic` if you want to use the default of the underlying type (i.e. `Either`, in which case it's default of its first type argument).

Reply:
Oh I see, I will just remove the None from there.